### PR TITLE
Fix UUID Output

### PR DESCRIPTION
**Pull request by bbernays**
Retain UUID type information when returning values from PGX
#303

### CODE REVIEW

File: pkg/policy/execute.go [hunk 1]
```diff
@@ -203,6 +212,23 @@ func (e *Executor) executeQuery(ctx context.Context, q *Query) (*QueryResult, er
...
+			if values[i] == nil {
+				continue
+			}
+			switch val.DataTypeOID {
+			case 2950:
+				fmt.Printf("val: %v\n", values[i])
+				t := uuid.FromBytesOrNil(ByteSlice(values[i].([16]uint8)))
+				fmt.Printf("Underlying Type: %T, %s\n", t, t.String())
+				values[i] = t
+			}
```
Comment (bbernays) [inline pkg/policy/execute.go:228]: I'm not sure if there is a more generic way where we wouldn't have to re-implement for every type

File: pkg/policy/execute.go [hunk 2]
```diff
@@ -203,6 +212,23 @@ func (e *Executor) executeQuery(ctx context.Context, q *Query) (*QueryResult, er
...
+
+		fmt.Printf("Underlying Type: %+v\n", values)
+		for i, val := range data.FieldDescriptions() {
+			// val.DataTypeOID
+			if values[i] == nil {
+				continue
+			}
+			switch val.DataTypeOID {
+			case 2950:
+				fmt.Printf("val: %v\n", values[i])
```
Comment (bbernays) [inline pkg/policy/execute.go:224]: This is in for local debugging/dev will be removed when pr is complete

File: pkg/client/uuid.go [hunk 3]
```diff
@@ -0,0 +1,273 @@
+package client
```
Comment (disq) [inline pkg/client/uuid.go:1]: If this was copied from pgtype, link to the original file + our PR maybe?

Comment (bbernays) [reply pkg/client/uuid.go:1]: Added a link to the file on Github.
Also added comments to the two places that changed `\\CQ-Change:`

Comment (bbernays) [reply pkg/client/uuid.go:1]: If this is not what you were looking for then let me know!

File: pkg/client/uuid.go [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+	"fmt"
+	"reflect"
+	"time"
+
+	"github.com/jackc/pgtype"
+)
+
+// Code came from: https://github.com/jackc/pgtype/blob/75446032b914bb0be5e07da29c976034c0a666cf/uuid.go#L3-L230
+
+type UUID struct {
```
Comment (roneli) [inline pkg/client/uuid.go:17]: Would embedding the original/calling the original code

**Review by roneli — CHANGES_REQUESTED** [review]
LGTM overall, had a general suggestion that can reduce the amount of code we have.

File: pkg/client/uuid.go (hunk 4 above)
Comment (bbernays) [reply pkg/client/uuid.go:9]: Great suggestion! Fixed!

**Review by disq — APPROVED** [review]

**Pull request merged**